CITATION.cff

```diff
@@ -0,0 +1,48 @@
+# SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model project <dynamic.grid.calculation@alliander.com>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
```

Comment:
Which parts have to be updated mannually?

Reply:
let's say there is a new author in the future (e.g. a new team member or contributor), then we would have to manually add it to this list, or create a Github action that verifies that this is still correct.

alternatively, we do not have a `CITATION.cff` file. Then `Zenodo` will automatically update the list of authors, but then you do not get the github `cite this` button.

i'm fine either way... your call

Reply:
ok :+1:

Reply:
@TonyXiang8787 is it correct that there are only 3 maintainers listed in the PGM project and that chenguang is one of you? https://github.com/orgs/PowerGridModel/teams/pgm-maintainers

Reply:
there are child team members (whatever that may mean) but chenguang is not in that child team member list

Reply:
All admins are also maintainers

Reply:
right, github is just doing a poor job at showing only the root maintainer members instead of the admin members

do you think it's a good idea to split the PGM_admins child team away from the child team so that outside collaborators can see that there are maintainers and admins in https://github.com/orgs/PowerGridModel/teams

![image](https://github.com/PowerGridModel/power-grid-model/assets/15234327/cacd1cd1-6385-48a9-ab87-51679a1f16da)
![image](https://github.com/PowerGridModel/power-grid-model/assets/15234327/2dc354b3-34a1-4932-83a3-9d860831009a)

Reply:
Let's keep this as separate issue.

Reply:
on that same note: i see that laurynas is not part of the maintainers nor admins

Reply:
@mgovers I will add him as maintainer.